SpriterDotNet/Spriter.cs

```diff
@@ -29,22 +29,16 @@ public class SpriterFolder : SpriterElement
     public class SpriterFile : SpriterElement
     {
         [XmlAttribute("pivot_x")]
-        public float PivotX;
+        public float PivotX = 0f;
```

Comment:
That's a bit of a bummer, but makes sense. Will revert this file and move things back to constructors.